media/css/newsletter/newsletter-firefox.less

```diff
@@ -0,0 +1,248 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import (reference) '../pebbles/lib.less';
+@import '../pebbles/components/footer.less';
+@import '../pebbles/elements/forms.less';
+
+@color-fox-purple: #3a304b;
+@color-link-blue: #00a7e0;
+
+.content {
+    .clearfix;
+
+    @media @mq-tablet {
+        width: @width-tablet;
+    }
+
+    @media @mq-desktop {
+        width: @width-desktop;
+    }
+}
+
+.section-intro {
+    background-color: @color-fox-purple;
+
+    .content {
+        background-image: url('/media/img/newsletter/firefox/mailbox.svg'),
+                          url('/media/img/newsletter/firefox/fox-tail.svg');
+        background-position: bottom -40px left 10px, bottom -45px right -10px;
+        background-repeat: no-repeat, no-repeat;
+        -webkit-background-size: 141px 157px, 153px 115px;
```

Comment:
Turns out a simple way to do this with Less is just to use a semicolon as the separator for the mixin value e.g. `.background-size(141px 157px, 153px 115px;);`. It looks a bit odd, but seems to be the recommended solution.

http://lesscss.org/features/#mixins-parametric-feature-mixins-with-multiple-parameters